siliconcompiler/tools/openroad/scripts/sc_procs.tcl

```diff
@@ -228,9 +244,25 @@ proc sc_supply_nets {} {
 
 proc sc_psm_check_nets {} {
   global openroad_psm_enable
+  global openroad_psm_skip_nets
 
   if { $openroad_psm_enable == "true" } {
-    return [sc_supply_nets]
+    set psm_nets []
+
+    foreach net [sc_supply_nets] {
+      set skipped false
+      foreach skip_pattern $openroad_psm_skip_nets {
+        if { [string match $skip_pattern $net] } {
+          set skipped true
+          break
+        }
+      }
+      if { !$skipped } {
+        lappend psm_nets $net
+      }
+    }
```

Comment:
Would something like this work? At least for full regexes.
For the `string match` I didn't find any pretty `lsearch` alternative...
```suggestion
    set joined_pattern [join $skip_nets "|"]
    foreach net [sc_supply_nets] {
      if { [regexp $joined_pattern $net] } {
        lappend psm_nets $net
      }
    }
```

Reply:
regexp and string match are somewhat different and globbing is more common in EDA